pkg/apis/kops/v1alpha2/zz_generated.conversion.go

```diff
@@ -1,7 +1,7 @@
 // +build !ignore_autogenerated
 
 /*
-Copyright 2019 The Kubernetes Authors.
+Copyright 2020 The Kubernetes Authors.
```

Comment:
These copyright years have already been updated on the master branch: https://github.com/kubernetes/kops/blob/master/pkg/apis/kops/v1alpha2/zz_generated.conversion.go any chance you can rebase master and then rerun `make apimachinery`? That should hopefully get rid of all these year updates which will make this PR much smaller.

Reply:
Done